.github/Contributors.md

```diff
@@ -18,6 +18,7 @@
 - [Mark Kuchel](https://github.com/kuchel77)
 - [Christian Nilsson](https://github.com/NiKiZe)
 - [Zhongxian Li](https://github.com/siriuslzx)
+- [Darsh Patel](https://github.com/darshkpatel) : Logo 🎉
```

Comment:
I believe I'm already on the contributor's list, Line number 9

Reply:
Haha, sorry about that.
I think that counts as a "Friday commit" or in this case minutes before hitting the pillow.

Reply:
Happens to the best of us!

Reply:
Fixed